readme.md

```diff
@@ -23,6 +23,8 @@ pip install -r requirements.txt
 pip install -e .
  ```
 
+If you are on Windows ensure that you have either MS-Office or LibreOffice installed in the default locations for now.
```

Comment:
You should probably clarify here what you mean with the 'default locations'. You are quite specific in the code, so maybe simply refer to that?